AppHandling/Start-NavContainerAppDataUpgrade.ps1

```diff
@@ -18,39 +18,64 @@ function  Start-BcContainerAppDataUpgrade {
     Param (
         [string] $containerName = $bcContainerHelperConfig.defaultContainerName,
         [string] $tenant = "default",
-        [Parameter(Mandatory=$true)]
+        [Parameter(Mandatory = $true)]
         [string] $appName,
         [string] $appVersion = "",
-        [string] $language = ""
+        [string] $language = "",
+        [string] $exclusiveAccessTicket = "",
+        [string] $path,

```

Comment:
I prefer defaulting parameters which are not mandatory.
Set path and syncmode to ""

Reply:
OK